ci/testnet-deploy.sh

```diff
@@ -59,19 +60,22 @@ EOF
   exit $exitcode
 }
 
-netName=$1
-cloudProvider=$2
-zone=$3
-[[ -n $netName ]] || usage
-[[ -n $cloudProvider ]] || usage "Cloud provider not specified"
-[[ -n $zone ]] || usage "Zone not specified"
-shift 3
+zone=()
 
-while getopts "h?p:Pn:c:t:gG:a:Dbd:rusx" opt; do
+while getopts "h?p:Pn:c:t:gG:a:Dbd:rusxz:N:C:" opt; do
   case $opt in
   h | \?)
     usage
     ;;
+  N)
```

Comment:
In other scripts we are using "p" to identify the network name. Maybe use that to keep the consistency.

Reply:
Done